bundle/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Executor.scala

```diff
@@ -32,20 +32,20 @@ object Step {
       q <- observe.leftMap(NonEmptyList(_))
     } yield StepResult(p, q)
 
-  def step(config: Config): (Set[System], Step) = {
+  def step(config: Config): SeqexecFailure \/ (Set[System], Step) = {
     val instName = config.getItemValue(new ItemKey(INSTRUMENT_KEY, INSTRUMENT_NAME_PROP))
     val instrument = instName match {
       case GmosSouth.name => Some(GmosSouth)
-      case Flamingos2.name => Some(Flamingos2(Flamingos2ControllerEpics))
+      case Flamingos2.name => Some(Flamingos2(Flamingos2ControllerSim))
```

Comment:
I guess these controls if we talk to the real systems or are in simulation, right?